index.html

```diff
@@ -118,7 +120,7 @@ <h3 class="header__title">
                     <img src="https://www.citypng.com/public/uploads/preview/-51609516149nowldl3ocn.png"
                         class="card-button__image" alt=""></img>
                     <div class="card-button__content-wrapper">
-                        <span class="card-button__content">
+                        <span class="card-button__content cantarell-regular">
```

Comment:
use a mixin instead of adding classes to every row. Remove all these classes for fonts and replace with the new way I showed you